### PR TITLE
[mesheryctl] add suggestions for `context switch` invalid arguments

### LINKED ISSUE #4806: [mesheryctl] UX: Need helpful suggestion to user of `system context switch`
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
When no parameters are provided, this command returns an error message that is not helpful.

```
$ mesheryctl system context switch
Error: accepts 1 arg(s), received 0
```

#### Desired Behavior
1. Show user example usage.
1. Show user help text for `system context switch`.

---

#### Contributor Resources

- ⌨️ [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by bisakhmondal**
Signed-off-by: Bisakh Mondal <bisakhmondal00@gmail.com>

**Description**

This PR fixes #
closes #4806

### CODE REVIEW

File: mesheryctl/internal/cli/root/system/context.go [hunk 1]
```diff
@@ -307,10 +307,19 @@ var viewContextCmd = &cobra.Command{
 
 // switchContextCmd represents the switch command
 var switchContextCmd = &cobra.Command{
-	Use:          "switch context-name",
-	Short:        "switch context",
-	Long:         `Configure mesheryctl to actively use one one context vs. the another context`,
-	Args:         cobra.ExactArgs(1),
+	Use:   "switch context-name",
+	Short: "switch context",
+	Long:  `Configure mesheryctl to actively use one one context vs. the another context`,
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/context.go:312]:
```suggestion
	Long:  `Configure mesheryctl to actively use one one context vs. another context`,
```

File: mesheryctl/internal/cli/root/system/context.go [hunk 2]
```diff
@@ -307,10 +307,19 @@ var viewContextCmd = &cobra.Command{
...
-	Short:        "switch context",
-	Long:         `Configure mesheryctl to actively use one one context vs. the another context`,
-	Args:         cobra.ExactArgs(1),
+	Use:   "switch context-name",
+	Short: "switch context",
+	Long:  `Configure mesheryctl to actively use one one context vs. another context`,
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = `Usage: mesheryctl system context switch [context name]
+Example: mesheryctl system context switch k8s-sample
+Description: It configures mesheryctl to actively use one one context vs. the another context`
```
Comment (bariqhibat) [inline mesheryctl/internal/cli/root/system/context.go:316]: I think it's more appropriate to remove the subject `It`, so just `Configures mesheryctl to actively...`

**Review by bariqhibat — COMMENTED** [review]
A comment

**Comment by leecalcote** [comment]
@bisakhmondal, it's great to see this PR. Some good work here. 👍

**Comment by leecalcote** [comment]
@bisakhmondal, I encourage you to jump into the Layer5 Slack (https://slack.layer5.io). Say, hi, and tell us what you're current focus is in APISIX.

**Review by leecalcote — APPROVED** [review]
Thanks, @bisakhmondal.

Do jump into the community and say hi. Perhaps, we should list APISIX on the service mesh landscape - https://layer5.io/landscape. What do you think?

**Pull request merged**